src/offazure/azext_offazure/generated/_params.py

```diff
@@ -0,0 +1,167 @@
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('offazure hyperv cluster list') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.')
+        c.argument('filter_', options_list=['--filter'], type=str, help='')
+
+    with self.argument_context('offazure hyperv cluster show') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.', id_part='subscription')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.', id_part='name')
+        c.argument('cluster_name', type=str, help='Cluster ARM name.', id_part='child_name_1')
+
+    with self.argument_context('offazure hyperv host list') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.')
+        c.argument('filter_', options_list=['--filter'], type=str, help='')
+
+    with self.argument_context('offazure hyperv host show') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.', id_part='subscription')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.', id_part='name')
+        c.argument('host_name', type=str, help='Host ARM name.', id_part='child_name_1')
+
+    with self.argument_context('offazure hyperv machine list') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.')
+        c.argument('filter_', options_list=['--filter'], type=str, help='')
+        c.argument('top', type=int, help='')
+        c.argument('continuation_token', type=str, help='Optional parameter for continuation token.')
+        c.argument('total_record_count', type=int, help='Total count of machines in the given site.')
+
+    with self.argument_context('offazure hyperv machine show') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.', id_part='subscription')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.', id_part='name')
+        c.argument('machine_name', type=str, help='Machine ARM name.', id_part='child_name_1')
+
+    with self.argument_context('offazure hyperv run-as-account list') as c:
+        c.argument('subscription_id', type=str, help='The ID of the target subscription.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('site_name', type=str, help='Site name.')
```

Comment:
the convention is to use `site` so it could accept both resource id and resource name right?

Reply:
Yeah, there is the convention to add the `id_part` in the argument, which makes the command support both `--name` and `--ids`, but the behavior appears on `show` and `delete` more often. For this case, i think `--site-name` is convenient enough.